response_operations_ui/views/collection_exercise.py

```diff
@@ -177,9 +179,17 @@ def view_collection_exercise(short_name, period):
         validation_failed=validation_failed,
         show_msg=show_msg,
         info_panel=info_panel,
+        total_ci_count=total_ci_count,
     )
 
 
+def get_total_ci_count(ci_table_context):
```

Comment:
The context builder already loops once, so we should try to do the logic in there

Reply:
Moved this logic inside the context builder